docs/using.rst

```diff
@@ -394,7 +396,12 @@ Running with Docker
 Docker_ is an amazingly simple and quick way to obtain a
 certificate. However, this mode of operation is unable to install
 certificates or configure your webserver, because our installer
-plugins cannot reach it from inside the Docker container.
+plugins cannot reach your webserver from inside the Docker container.
+
+Most users should use the operating system packages (available from 
+certbot.eff.org_) or, as a fallback, ``certbot-auto``. You should only 
```

Comment:
So the packages aren't actually available on https://certbot.eff.org. However, that website gives you instructions how to obtain those packages from your operating system.